eclass/distutils-r1.eclass

```diff
@@ -561,6 +561,20 @@ distutils_enable_tests() {
 				test_pkgs+=' dev-python/pytest-xdist[${PYTHON_USEDEP}]'
 			fi
 
+			local plugin
+			_set_epytest_plugins
+			for plugin in "${EPYTEST_PLUGINS[@]}"; do
+				case ${plugin} in
```

Comment:
Should we have an associative array for these, especially given it's done in global-scope?

Reply:
The mapping, sorry.

Reply:
I don't think we'd be able to use the patterns then, though.